knees4bees/whats-cookin
# Feature/recipe class and tests

--->  ((are you in the right branch??))
## What does this PR do?
 - Creates a `Recipe` class and associated tests.
 
## How should it be tested?
 - Look over `Recipe.js` and `Recipe-test.js`: does anything look ill-advised?
 - Run `npm test` in the terminal and confirm that all test pass
 - Review spec for Recipe class and 
   - confirm that all functionality is present (note that right now there are not explicit methods for returning a recipe's ingredient names or instructions; these are properties of the object that can be accessed with `myRecipe.name` or `myRecipe.ingredients[0].name`)
   - confirm that all properties/methods are being tested
